src/consensus/sub_block_record.py

```diff
@@ -52,6 +55,13 @@ class SubBlockRecord(Streamable):
     # Sub-epoch (present iff this is the first SB after sub-epoch)
     sub_epoch_summary_included: Optional[SubEpochSummary]
 
+    # todo remove the hash lists (redundant)
```

Comment:
Can you do # TODO(almog) or # TODO(mariano): so we can look it up later

Reply:
sure